compiler/src/iree/compiler/Codegen/Common/test/tile_and_distribute_to_workgroups.mlir

```diff
@@ -1726,3 +1728,92 @@ hal.executable private @no_tile {
 // CHECK-LABEL: func @no_tile()
 //   CHECK-NOT: scf.for
 //       CHECK: iree_linalg_ext.topk
+
+// -----
+
+hal.executable private @pack_lowering {
+  hal.executable.variant public @embedded_elf_x86_64, target = <"llvm-cpu", "embedded-elf-x86_64", {}> {
+    hal.executable.export public @gemm_lhs_pack ordinal(0) 
+        layout(#hal.pipeline.layout<push_constants = 0,
+            sets = [<0, bindings = [<0, storage_buffer, ReadOnly>, <1, storage_buffer>]>]>)
+        attributes {translation_info = #iree_codegen.translation_info<CPUDataTiling>} {
+    ^bb0(%arg0: !hal.device, %arg1: index, %arg2: index):
+      %x, %y, %z = flow.dispatch.workgroup_count_from_set_encoding_op %arg1, %arg2
+      hal.return %x, %y, %z : index, index, index
+    }
+    builtin.module {
+      func.func @gemm_lhs_pack() {
+        %c0 = arith.constant 0 : index
+        %cst = arith.constant 0.000000e+00 : f32
+        %0 = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) offset(%c0) alignment(64)
+            : !flow.dispatch.tensor<readonly:tensor<100x250xf32>>
+        %1 = hal.interface.binding.subspan set(0) binding(1) type(storage_buffer) offset(%c0) alignment(64)
+            : !flow.dispatch.tensor<writeonly:tensor<14x64x8x4xf32>>
+        %2 = flow.dispatch.tensor.load %0, offsets = [0, 0], sizes = [100, 250], strides = [1, 1]
+            : !flow.dispatch.tensor<readonly:tensor<100x250xf32>> -> tensor<100x250xf32>
+        %3 = tensor.empty() : tensor<14x64x8x4xf32>
+        %4 = iree_linalg_ext.pack {lowering_config = #iree_codegen.lowering_config<tile_sizes = [[64, 64]]>} %2
+            padding_value(%cst : f32) inner_dims_pos = [0, 1] inner_tiles = [8, 4] into %3
+            : (tensor<100x250xf32> tensor<14x64x8x4xf32>) -> tensor<14x64x8x4xf32>
+        flow.dispatch.tensor.store %4, %1, offsets = [0, 0, 0, 0], sizes = [14, 64, 8, 4], strides = [1, 1, 1, 1]
```

Comment:
Will we need a pattern to replace `%3` with `%1` in the pack op in ConvertToDestinationPassingStylePass? I feel that there will be a stack allocation if tensor.empty() is used.

Reply:
Ill see if that is the case.... If needed will do it as a follow up, but the `LinalgExt` ops should be handled like all other `LinalgExt` ops.